src/swift/Data.swift

```diff
@@ -156,19 +193,41 @@ public struct DispatchData : RandomAccessCollection {
 	/// - parameter pointer: A pointer to the buffer you wish to copy the bytes into.
 	/// - parameter count: The number of bytes to copy.
 	/// - warning: This method does not verify that the contents at pointer have enough space to hold `count` bytes.
+	@available(swift, deprecated: 4, message: "Use copyBytes(to: UnsafeMutableRawBufferPointer, count: Int) instead")
 	public func copyBytes(to pointer: UnsafeMutablePointer<UInt8>, count: Int) {
 		_copyBytesHelper(to: pointer, from: 0..<count)
 	}
+
+	/// Copy the contents of the data to a pointer.
+	///
+	/// - parameter pointer: A pointer to the buffer you wish to copy the bytes into.
+	/// - parameter count: The number of bytes to copy.
+	/// - warning: This method does not verify that the contents at pointer have enough space to hold `count` bytes.
```

Comment:
Stepping back, does it also make sense to default the `count` to the entire data, or would someone just not write that? How do I do an offset?

Stepping back even further…is this a necessary API at all? Normally in Swift we'd write this as

```
buffer.append(data[0..<count])
```

and if that's not going to be as efficient we should try to figure out what to do there. cc @airspeedswift